doc/devguide/extending/app-layer/img/generate-images.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Script to generate Sequence Diagram images with mscgen
+#
+
+cd extending/app-layer/img
+
+for FILE in *.msc ; do
+    # call mscgen and convert each file in images dir
+    mscgen -T png -F Arial $FILE
+    # if command fails, lets inform about that
+    if [ $? -ne 0 ]; then
+        echo "$FILE couldn't be converted in the devguide"
```

Comment:
Should we not exit there ?

Reply:
I'm not sure about that, what if that was just with a specific file, because of bad naming, for instance?

Reply:
This looks like not supposed to happen, so let's stop the work so that the user notices ;-)

Reply:
That makes way more sense 😅  sorry for sounding stubborn sometimes. Patched.